### PR TITLE
Remove getOwnHashObject

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

**Comment by arv** [comment]
@johnjbarton PTAL

File: src/runtime/runtime.js [hunk 1]
```diff
@@ -27,12 +27,10 @@
   var $TypeError = TypeError;
   var $create = $Object.create;
   var $defineProperty = $Object.defineProperty;
-  var $freeze = $Object.freeze;
+  var {freeze} = $Object;
```
Comment (johnjbarton) [inline src/runtime/runtime.js]: It's not clear to me why this one is `freeze` while all other functions of `$Object` are eg `$freeze`.

Comment (arv) [inline src/runtime/runtime.js]: I was thinking I would use destructuring to save some characters... not worth it. Changing back.

**Comment by arv** [comment]
PTAL

**Comment by arv** [comment]
TBR

Going to land this... You LGTMed a few PRs that depended on this.

**Pull request merged**